spring-xsuaa/pom.xml

```diff
@@ -48,6 +48,10 @@
 			<groupId>${project.groupId}</groupId>
 			<artifactId>api</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
as of now, we can not improve the spring-xsuaa code, as the Token creation logic is not yet available in java-api. That's why I would undo that change.